### DISCUSSION TITLE
[Task]: Onnx to tensorrt conversion

### DISCUSSION THREAD

**Discussion by MoaazAbdulrahman** [open-edge-platform/anomalib#847] [Q&A]
### What is the motivation for this task?

Hi,

I am trying to convert the output onnx model to tensorrt usig trtexec. I have tried PatchCore and PADIM but I am getting the following error. 

`ERROR: onnx2trt_utils.cpp:188 In function convertAxis:
[8] Assertion failed: axis >= 0 && axis < nbDims`

Is there anyway to solve this issue. 
Thanks

### Describe the solution you'd like

If there is any script to export the onnx model to tensorrt that will be nice.

**Comment by samet-akcay** [comment]
@MoaazAbdulrahman, tensortrt support is currently beyond our scope, so not sure what a solution would be like. I'll be moving this to discussions so we could hopefully get some help from the community. Thanks!

**Comment by laborer123** [comment]
@MoaazAbdulrahman Have you tried other models that can be converted？I tried torch2trt to convert fastflow and failed
